src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/search.html.twig

```diff
@@ -1,11 +1,11 @@
 <div id="sidebar-search">
     <form action="{{ path('_profiler_search') }}" method="get">
-        <div class="form-group">
+        <div class="form-group row">
```

Comment:
this does not make sense. The WebProfilerBundle does not use Bootstrap, but its own styles